core/src/main/java/com/linecorp/armeria/server/streaming/JsonTextSequences.java

```diff
@@ -149,6 +151,21 @@ public static HttpResponse fromPublisher(ResponseHeaders headers, Publisher<?> c
         return streamingFrom(contentPublisher, sanitizeHeaders(headers), trailers, o -> toHttpData(mapper, o));
     }
 
+    /**
+     * Creates a new JSON Text Sequences from the specified {@link Publisher}.
+     *
+     * @param headers the HTTP headers supposed to send
+     * @param contentPublisher the {@link Publisher} which publishes the objects supposed to send as contents
+     * @param trailers the HTTP trailers
+     * @param contentConverter the function which converts the content object into a UTF-8 JSON string.
+     */
+    public static HttpResponse fromPublisher(ResponseHeaders headers, Publisher<?> contentPublisher,
+                                             HttpHeaders trailers, Function<Object, String> contentConverter) {
```

Comment:
```suggestion
    public static <T> HttpResponse fromPublisher(ResponseHeaders headers, Publisher<T> contentPublisher,
                                             HttpHeaders trailers, Function<T, String> contentConverter) {
```

Does this work (for all)?

Reply:
That sounds good.